README.md

```diff
@@ -1,30 +1,26 @@
-# Front-End Repo for Node/React Practicum
+## Description
 
-This will be the front-end for your team's practicum project.
+The Music App is a React application where users can stream music, add songs to wishlist and select and buy songs.
 
-It is suggested that you run these instructions **after** you setup the back-end server first.
-You can go through these steps during your first group meeting in case you need assistance from your mentors.
 
-You will have two folders inside one team folder (one for front-end and one for back-end). Name the parent folder something appropriate (in the below example we title it "Practicum Project"). Then clone directly (do not fork and clone) the front and back repos while inside the parent ("Practicum Project") project folder.
+## Screenshots
 
-![folders](images/folder_structure.png)
+![Signin](images\signin.png)
 
-> The front-end app (React) will be running on port 3000. The back-end server will be running on port 8000. You will need to run both the front-end app and the back-end server at the same time to test your app.
+![Home](images\homeapp)
 
-### Setting up local development environment
+## Technology Stack
```

Comment:
We can also mention that this is a `React` app with use of:

*   Styled-components together with material-ui
*   Redux Toolkit for app-wide state management
*   Axios for HTTP requests
*   React-Router for client-side routing
*   Jest for testing
*   Mirage.js for mocking requests for dev/test environments

And also some local dev tooling and continuous integration (CI) and continuous deployment (CD) tooling:

*   Husky + Lint-staging for ensuring linting and formatting happens on each commit
*   Prettier for consistent formatting 
*   ESLint for catching syntax errors
*   Github Actions for running linting and formatting on each PR push.